torch/csrc/lazy/core/shape_inference.h

```diff
@@ -11,59 +11,59 @@
-TORCH_API std::vector<Shape> compute_shape_native_layer_norm_backward(const at::Tensor & grad_out, const at::Tensor & input, at::IntArrayRef normalized_shape, const at::Tensor & mean, const at::Tensor & rstd, const c10::optional<at::Tensor> & weight, const c10::optional<at::Tensor> & bias, ::std::array<bool,3> output_mask);
-TORCH_API std::vector<Shape> compute_shape_nll_loss2d_backward(const at::Tensor & grad_output, const at::Tensor & self, const at::Tensor & target, const c10::optional<at::Tensor> & weight, int64_t reduction, int64_t ignore_index, const at::Tensor & total_weight);
-TORCH_API std::vector<Shape> compute_shape_nll_loss2d_forward(const at::Tensor & self, const at::Tensor & target, const c10::optional<at::Tensor> & weight, int64_t reduction, int64_t ignore_index);
-TORCH_API std::vector<Shape> compute_shape_random_(at::Tensor & self, c10::optional<at::Generator> generator);
-TORCH_API std::vector<Shape> compute_shape_random_(at::Tensor & self, int64_t to, c10::optional<at::Generator> generator);
-TORCH_API std::vector<Shape> compute_shape_random_(at::Tensor & self, int64_t from, c10::optional<int64_t> to, c10::optional<at::Generator> generator);
-TORCH_API std::vector<Shape> compute_shape_relu(const at::Tensor & self);
-TORCH_API std::vector<Shape> compute_shape_relu_(at::Tensor & self);
-TORCH_API std::vector<Shape> compute_shape_repeat(const at::Tensor & self, at::IntArrayRef repeats);
-TORCH_API std::vector<Shape> compute_shape_smooth_l1_loss_backward(const at::Tensor & grad_output, const at::Tensor & self, const at::Tensor & target, int64_t reduction, double beta);
-TORCH_API std::vector<Shape> compute_shape_sort(const at::Tensor & self, int64_t dim, bool descending);
-TORCH_API std::vector<Shape> compute_shape_stack(at::TensorList tensors, int64_t dim);
-TORCH_API std::vector<Shape> compute_shape_std(const at::Tensor & self, bool unbiased);
-TORCH_API std::vector<Shape> compute_shape_std(const at::Tensor & self, at::IntArrayRef dim, bool unbiased, bool keepdim);
-TORCH_API std::vector<Shape> compute_shape_std(const at::Tensor & self, at::OptionalIntArrayRef dim, c10::optional<int64_t> correction, bool keepdim);
-TORCH_API std::vector<Shape> compute_shape_sum(const at::Tensor & self, c10::optional<at::ScalarType> dtype);
-TORCH_API std::vector<Shape> compute_shape__to_copy(const at::Tensor & self, c10::optional<at::ScalarType> dtype, c10::optional<at::Layout> layout, c10::optional<at::Device> device, c10::optional<bool> pin_memory, bool non_blocking, c10::optional<at::MemoryFormat> memory_format);
-TORCH_API std::vector<Shape> compute_shape_trace(const at::Tensor & self);
-TORCH_API std::vector<Shape> compute_shape_zero_(at::Tensor & self);
-TORCH_API std::vector<Shape> compute_shape_narrow_copy(const at::Tensor & self, int64_t dim, int64_t start, c10::SymInt length);
+TORCH_API std::vector<torch::lazy::Shape> compute_shape__adaptive_avg_pool2d(const at::Tensor & self, at::IntArrayRef output_size);
```

Comment:
haha. ok, we may have a namespace war on our hands

Reply:
(this is ok to change to lazy:: IMO.  but.. i also worry a little bit that people will not realize it breaks something when they change it back as a cleanup)

Reply:
I think as long as that pr trigger a xla test they will see xla build fail.